### PR TITLE
Symbols tint

### PR DESCRIPTION
**Pull request by milg15**
Basically reuses the fillInspector to work with tints and I also added some correction where selectin 'No Symbol' in the Override wasn't working

image
image

### CODE REVIEW

File: packages/app/src/containers/FillInspector.tsx [hunk 1]
```diff
@@ -9,7 +9,13 @@ import {
 } from '../contexts/ApplicationStateContext';
 import useShallowArray from '../hooks/useShallowArray';
 
-export default memo(function FillInspector() {
+export default memo(function FillInspector({
+  title = 'Fill',
```
Comment (dabbott) [inline packages/app/src/containers/FillInspector.tsx:13]: Let's remove the default values and pass them into both `<FillInspector />`s. I think it'll be clearer to be explicit here

**Review by dabbott — APPROVED** [review]
👍 Looks great. So the tint is stored as a fill in the sketch file data 🤔. Interesting!

File: packages/app/src/containers/FillInspector.tsx (hunk 1 above)
Comment (milg15) [reply packages/app/src/containers/FillInspector.tsx:13]: okey!

**Comment by milg15** [comment]
yes I thought it would be somewhere else but It was just there 😄

**Pull request merged**